✨ feat: improve logo design

okineadev opened the pull request:
Filled the arrow in the logo with white color so that it is better displayed in the marketplace

Also renamed `res` folder with logos to `public`

okineadev committed:
feat: :art: improve logo design

yunsii commented:
> Filled the arrow in the logo with white color so that it is better displayed in the marketplace

Good job, thinks :smile:

> Also renamed `res` folder with logos to `public`

It is not necessary to rename `res` folder, the related options should be also updated if so https://github.com/yunsii/vscode-links/blob/dabcbbcc89777208d005900e881429bb3b0efd69/package.json#L27-L31

In the end, just filled the arrow is good enough.

okineadev committed:
fix: :bento: revert `res` folder name

okineadev commented:
> > Filled the arrow in the logo with white color so that it is better displayed in the marketplace
> 
> Good job, thinks 😄
> 
> > Also renamed `res` folder with logos to `public`
> 
> It is not necessary to rename `res` folder, the related options should be also updated if so
> 
> https://github.com/yunsii/vscode-links/blob/dabcbbcc89777208d005900e881429bb3b0efd69/package.json#L27-L31
> 
> In the end, just filled the arrow is good enough.

Fixed
